README.md

```diff
@@ -70,6 +70,14 @@ dependencies {
 }
```

Comment:
Dont forget to update version number in README (near the top, with the gradle example)